app/models/miq_queue.rb

```diff
@@ -123,6 +123,66 @@ def self.put(options)
     msg
   end
 
+  # Trigger a background job
+  #
+  # target_worker:
+  #
+  # @options options [String] :class
+  # @options options [String] :instance
+  # @options options [String] :method
+  # @options options [String] :args
+  # @options options [String] :target_id (deprecated)
```

Comment:
We use in 3 spots. 2 can be changed. Smart state host scanning uses this as a driver and it is more complicated.